client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/ContactForm/ContactForm.tsx

```diff
@@ -129,17 +129,20 @@ const ContactForm: React.FC<Props> = ({ updatedContactIndex, contactStatus, cont
                                             <Select
                                                 disabled={isFieldDisabled}
                                                 test-id='contactType'
-                                                defaultValue={Array.from(contactTypes.keys())[0]}
+                                                defaultValue={Array.from(contactTypes.keys())[2]}
```

Comment:
why 2 and not 1?

Reply:
maybe it is better to put the number on const?